test/test_api.py

```diff
@@ -290,15 +291,20 @@ def test_visual_search(self, mocker):
         """ should allow using visual search """
         mocker.return_value = MOCK_RESPONSE
 
-        api.visual_search(REMOTE_TEST_IMAGE, API_TEST_ASSET_ID, "sample image")
+        api.visual_search(REMOTE_TEST_IMAGE, API_TEST_ASSET_ID, "sample image", TEST_IMAGE)
```

Comment:
I'm fairly new to python, so asking out of curiosity:
- the change in the interface of `visual_search` was that you added a new named parameter called `image_file`
- so, the method can be used like this:
```python
api.visual_search(image_file=file)
```
or like here in the line 294?

If so, then aren't we breaking an existing contract? Before the PR, the fourth argument was `options`, now it's a file handler or path to a local file.

Also, I think that this line in the tests might suggest that it's the correct usage of the method, while the documentation says that only **one** of those four options is a correct one.